src/AtelierTomato.Markov.Storage.Sqlite/Model/AuthorPermissionRow.cs

```diff
@@ -6,8 +6,8 @@ public class AuthorPermissionRow
 	{
 		public string Author { get; set; }
 		public string QueryScope { get; set; }
-		public string AllowedScope { get; set; }
-		public AuthorPermissionRow(string author, string queryScope, string allowedScope)
+		public string? AllowedScope { get; set; }

```

Comment:
i still find the null to be needlessly confusing + we are not documenting what the null does
if it means all: i am sure OID has a way to voice all without using null
if it means none: this should be commented

Reply:
functionally can't handle it with our current OID setup because the base level is Service, we would have to prepend something to it or something it just unexists. we can think about changing it later